projects/cdk/services/meta-color.service.ts

```diff
@@ -0,0 +1,44 @@
+import type {OnDestroy} from '@angular/core';
+import {inject, Injectable} from '@angular/core';
+import {WA_WINDOW} from '@ng-web-apis/common';
+import {Subject} from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class TuiMetaColorService implements OnDestroy {
```

Comment:
Maybe better call it `TuiThemeColorService`?

Reply:
fixed